app/services/course/assessment/question/programming_import_service.rb

```diff
@@ -0,0 +1,105 @@
+# Imports the provided programming package into the question. This evaluates the package to
+# obtain the set of tests, as well as extracts the templates from the package to be stored
+# together with the question.
+class Course::Assessment::Question::ProgrammingImportService
```

Comment:
If I understand correctly, this service is for importing the test cases and store them in questions ? 

But when will this service be called?  And can we modify the test cases in questions ?  I'm a little bit confused.